src/index.scss

```diff
@@ -1,3 +1,12 @@
-body {
-  font-family: "Roboto";
+@import url('https://fonts.googleapis.com/css?family=Roboto+Condensed:300,700');
+@import url('https://fonts.googleapis.com/icon?family=Material+Icons');
+
+@import './theme.scss';
+
+html {
+  body {
+    margin: 0;
+    font-family: 'Roboto Condensed', sans-serif, monospace;
+    background: $light-gray;
+  }
 }
```

Comment:
add new line ad the end of file

Reply:
Since there is no warning showing up regarding EOL, there's already exists EOL on these files.